cmd/genconversion/conversion.go

```diff
@@ -92,17 +92,25 @@ func main() {
 		glog.Fatalf("Error while writing package line: %v", err)
 	}
 
+	var schemeVersion string
+	if *groupVersion == "" || *groupVersion == "/" {
+		// This occurs when we generate deep-copy for internal version.
+		schemeVersion = ""
+	} else {
+		schemeVersion = *groupVersion
```

Comment:
> the trailing slash on internal versions of grouped APIs doesn't cause issues?

That is the proper serialization.  `group/version` with version="", produces an ambiguous `group/`